packages/driver/test/cypress/integration/commands/assertions_spec.coffee

```diff
@@ -1,6 +1,8 @@
 $ = Cypress.$.bind(Cypress)
 _ = Cypress._
 
+logger = require('../../../../../runner/src/lib/logger')
```

Comment:
hmm, can we use `@packages/runner` instead of `../../../` infinity here?

Reply:
could get the symlink to work, even after `npm run link`